deps/undici/src/lib/core/diagnostics.js

```diff
@@ -6,6 +6,12 @@ const undiciDebugLog = util.debuglog('undici')
 const fetchDebuglog = util.debuglog('fetch')
 const websocketDebuglog = util.debuglog('websocket')
 let isClientSet = false
+let tracingChannel
+
+if (diagnosticsChannel.tracingChannel) {
+  tracingChannel = diagnosticsChannel.tracingChannel('undici:fetch')
```

Comment:
```suggestion
  fetchTracingChannel = diagnosticsChannel.tracingChannel('undici:fetch')
```

maybe?

Reply:
Oh, I think you are using this nomenclature because it should match what exists in deps/undici?

Reply:
Same